### PR TITLE
ImageSequence - New Component

### LINKED ISSUE #31: [Component] - Image Sequence
Generic solution - Scroll based.

Features:
Load pivot images: load every 4th image at first and fill the gaps later

### PR DESCRIPTION
**Pull request by artemleznikov**
#31 
Image Sequence component which draws image frames onto a canvas based on scroll percentage.

Features:
- Initially loads every fourth image, then fills gaps after render
- Replicates css property `background-size: cover` with background position when drawing images
- Tooltips with advanced positioning which stick to a pinned point on the canvas
- Has fallback logic to load a substitute image and waiting for current active image to load before drawing

### CODE REVIEW

File: src/components/ImageSequence/ImageSequence.js [hunk 1]
```diff
@@ -0,0 +1,244 @@
+import React, { memo, useLayoutEffect, useRef, useCallback, useEffect, useState, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import checkProps from '@jam3/react-check-extra-props';
+
+import './ImageSequence.css';
```
Comment (namniak) [inline src/components/ImageSequence/ImageSequence.js:6]:
```suggestion
import './ImageSequence.scss';
```

Comment (namniak) [inline src/components/ImageSequence/ImageSequence.js:6]:
```suggestion
import './ImageSequence.scss';
```
should be supported now

File: src/components/ImageSequence/ImageSequence-story.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      percentVisibleStart: 60,
+      percentVisibleEnd: 85,
+      content: 'I am tooltip three!'
+    }
+  ],
+  percentDrawOffsetX: 50,
+  percentDrawOffsetY: 50
+};
+
+storiesOf('ImageSequence', module).addWithJSX('Default', () => <Component {...props} />);
```
Comment (realdeemy) [inline src/components/ImageSequence/ImageSequence-story.js]:
```suggestion
storiesOf('ImageSequence', module).add('Default', () => <Component {...props} />);
```

File: src/components/ImageSequence/ImageSequence.js (hunk 1 above)
Comment (realdeemy) [inline src/components/ImageSequence/ImageSequence.js]:
```suggestion
import './ImageSequence.scss';
```

File: src/components/ImageSequence/ImageSequence.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
+.ImageSequence {
+  // Default height, overwritten in JS by heightMultiplier and images length
+  height: 300vh;
+
+  // Remove this: test code used to demonstrate sticky functionality within Storybook
+  margin-top: 50vh;
+  margin-bottom: 50vh;
```
Comment (realdeemy) [inline src/components/ImageSequence/ImageSequence.scss]: let's set these withing `story` file?

**Review by realdeemy — APPROVED** [review]
@artemleznikov amazing!
left minor comments

also pls rebase

**Pull request merged**